Fakebook.Posts/Fakebook.Posts.DataAccess/Repositories/PostsRepository.cs

```diff
@@ -144,6 +144,23 @@ public IEnumerator<Post> GetEnumerator()
 
         IEnumerator IEnumerable.GetEnumerator() => GetEnumerator();
 
+
+        public async Task<List<Post>> GetRecentPostsAsync(string userEmail, int recentInMinutes, DateTime dateNow)
+        {
+
+            var latestTime = new DateTimeOffset(dateNow - TimeSpan.FromMinutes(recentInMinutes));
+            
+
+            var query = await _context.Posts
+                .Where(u => u.UserEmail == userEmail)
+                .Where(t => DateTimeOffset.Compare(t.CreatedAt, latestTime) >= 0)
+                .ToListAsync();
+            var queryResult = query.Select(s => s.ToDomain());
+            return queryResult.ToList();
+        }
+        
+
+
```

Comment:
why so many random empty lines added all over this PR? it's just sloppy. i'll let it through except for that one file

Reply:
ok :(